src/api/movie-api.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const BASE_URL = 'http://react-cdp-api.herokuapp.com/movies'
```

Comment:
"movies" is considered an endpoint and could be a subject to change in the app lifecycle.
Consider extracting this endpoint details and putting it as MOVIES_ENDPOINT for example under config directory.